client/dashboard/me/billing-purchases/dataviews.tsx

```diff
@@ -250,20 +281,12 @@ export function getFields( {
 			render: ( { item }: { item: Purchase } ) => {
 				const isTransferred = isTransferredOwnership( item.ID, transferredPurchases );
 				return (
-					<div>
-						{ isTransferred ? (
-							getTitleForDisplay( item ) + '&nbsp;'
-						) : (
-							<Link
-								to={ purchaseSettingsRoute.fullPath }
-								params={ { purchaseId: item.ID } }
-								title={ __( 'Manage purchase' ) }
-							>
-								{ getTitleForDisplay( item ) }
-							</Link>
-						) }
+					<HStack justify="flex-start" spacing={ 1 }>
+						<PurchaseSettingLink purchase={ item } isTransferred={ isTransferred }>
```

Comment:
I wonder if this would be more readable if we made it explicit that `isTransferred` means `disabled`?

(Then we can remove the special logic in that component.)

```suggestion
						<PurchaseSettingLink purchase={ item } disabled={ isTransferred }>
```

Also, we're not likely to show a link to a transferred purchase anywhere else except the purchases list.

Reply:
Done by https://github.com/Automattic/wp-calypso/pull/106827/commits/6c9409c3a33237bae607e1542002555a6a5bb18e